skops/hub_utils/_hf_hub.py

```diff
@@ -737,8 +743,12 @@ def get_model_output(repo_id: str, data: Any, token: Optional[str] = None) -> An
     Also note that if the model repo is private, the inference API would not be
     available.
     """
-    # TODO: the "type: ignore" should eventually become unncessary when hf_hub
-    # is updated
+    warnings.warn(
+        "This feature is no longer free on hf.co and therefore this function will"
```

Comment:
Technically speaking, it's still free, just heavily rate-limited.

Reply:
So it's not free 😁

Reply:
A philosophical question...